pages.id/windows/winget.md

```diff
@@ -0,0 +1,39 @@
+# winget
+
+> Manajer Paket Antarmuka Baris Perintah Windows.
+
+> Informasi lebih lanjut: https://learn.microsoft.com/id-id/windows/package-manager/winget.
+
+- Instal aplikasi:
+
+`winget install {{namaaplikasi}}`
```

Comment:
```suggestion
`winget install {{nama_aplikasi}}`
```

And for subsequent mentions of `namaaplikasi`.

Reply:
Also, there seem to be differences between the commands on this page and the English page can you check it?

Reply:
In English version it is called as an _package_, and the [Indonesian translation of `apt`](https://nix.reinhart1010.id/id/linux/apt.html) uses `{{nama_paket}}` instead of `{{nama_aplikasi}}`.

Maybe should we standardize it?

```suggestion
`winget install {{nama_paket}}`
```

Reply:
This is the difference that I am talking about between this page and the main page.

![image](https://user-images.githubusercontent.com/26346867/193750495-8c50bd37-2f05-47d8-9bc8-4fd77661c081.png)

Should we update the [main page](https://github.com/tldr-pages/tldr/blob/main/pages/windows/winget.md) after this PR?